Source/devices/oni/AcqBoardONI.h

```diff
@@ -306,9 +306,13 @@ class AcqBoardONI : public AcquisitionBoard
     static constexpr double v3AdcBitVal = double ((((1.25 * (1 + 84.5 / 51)) / (1 << 12)) * (10 / (1.25 * (1 + 84.5 / 51)))) / 16);
 
     static constexpr int NUMBER_OF_PORTS = 4;
-    static constexpr int BNO_CHANNELS = 4;
+    static constexpr int BNO_CHANNELS = 3 + 3 + 4 + 3 + 1 + 1;
     static constexpr int MEMORY_MONITOR_FS = 100;
 
+    static constexpr double eulerAngleScale = 1.0f / 16;
```

Comment:
If you already had this info, why was it not used in the channel info bitVolts value?
bitVolts are, just, what 1 LSB means in terms of the unit. Just the inverse of the scale